web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts

```diff
@@ -141,15 +138,14 @@ export class EditFlowComponent implements OnInit, OnDestroy {
     });
   }
   runFlow(runObject): void {
-    this.runFlowClicked = true;
     this.manageFlowsService.runFlow(runObject).subscribe(resp => {
-      // TODO add error handling for error response, set runFlowClicked = false;
+      // TODO add response check
+      console.log('run flow resp', resp);
```

Comment:
Not sure of the team standards yet, but I usually vote on removing log statements like this